scripts/bad-linebreaks.mjs

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+import fs from 'fs';
+import { glob } from 'glob';
+
+function getLine(txt, index) {
+
+  let line = 1;
+
+  for (let i = 0; i < index; i++) {
+    if (txt[i] === '\n') {
+      line += 1;
+    }
+  }
+
+  return line;
+
+}
+
+export function findBadLinebreaks(file, fix = false) {
+
+  let contents = fs.readFileSync(file, 'utf8').toString();
+
+  const re = /(?<=[\w\d ])\n(?=[\w\d])/g;
+  const matches = Array.from(contents.matchAll(re));
+
+  for (const m of matches) {
+
+    if (fix) {
+      contents = `${contents.substring(0, m.index)} ${contents.substring(m.index + 1)}`;
+    }
+    else {
```

Comment:
```suggestion
    } else {
```
elses need cuddles

Reply:
i was serious about this one, it's the overwhelmingly idiomatic style for JS

Reply:
weird hill to die on

Reply:
you get brace style, I keep my emotional support whitespace